llama_stack/apis/resource.py

```diff
@@ -14,6 +14,8 @@ class ResourceType(Enum):
     shield = "shield"
     vector_db = "vector_db"
     dataset = "dataset"
+    grader = "grader"
+    # TODO: migrate scoring_function -> grader
```

Comment:
can we add deprecation warnings into scoring_functions as part of this PR?

Reply:
Let's keep this PR for API only for now, and add deprecation later with implementation fix.